app/models/label/LabelValidationTable.scala

```diff
@@ -177,6 +178,32 @@ object LabelValidationTable {
     }.list
   }
 
+  /**
+    * @return count of validations for the given label type
+    */
+  def countValidationsByLabelType(labelType: String): Int = db.withSession { implicit session =>
+    val typeID = LabelTypeTable.labelTypeToId(labelType)
+
+    val lv = for {
+      (v, l) <- labelValidationTable.innerJoin(labelsWithoutDeleted).on(_.labelId === _.labelId)
+    } yield l.labelTypeId
+
+    lv.list.count(x => x == typeID)
```

Comment:
We are going to want to do this counting using Slick. I believe what your code does right now is read in the entire list of validations from the database and put them into a list of Ints in Scala, then they are counted. Given that we are going to end up having millions of validations, I think it is best to keep this query all in Slick.

I think it would just be something like
```
def countValidationsByLabelType(labelType: String): Int = db.withSession { implicit session =>
  val typeID = LabelTypeTable.labelTypeToId(labelType)

  labelValidationTable.innerJoin(labelsWithoutDeleted).on(_.labelId === _.labelId)
    .filter(_._2.labelTypeId === typeID)
    .size
}
```